requirements/requirements_nlp.txt

```diff
@@ -21,3 +21,4 @@ rapidfuzz
 rouge_score
 sacrebleu[ja]
 sentence_transformers
+git+https://github.com/NVIDIA/lddl.git
```

Comment:
There's no pip install?

Reply:
this is how the pip install is specified in the main package https://github.com/NVIDIA/LDDL.